navigation-compose/src/main/kotlin/com/freeletics/khonshu/navigation/compose/NavHost.kt

```diff
@@ -61,6 +61,44 @@ public fun NavHost(
     deepLinkPrefixes: Set<DeepLinkHandler.Prefix> = emptySet(),
     navController: NavHostController = rememberNavController(),
     destinationChangedCallback: ((BaseRoute) -> Unit)? = null,
+) {
+    NavHost(
+        startRoute = startRoute,
```

Comment:
This is currently a recursive call and not calling the other composable since the shared base type is `BaseRoute`.

Reply:
Fixed in [b4c242c](https://github.com/freeletics/khonshu/pull/603/commits/b4c242ceabebf09688676db92ce92d3a04304bc1)

Reply:
:man_facepalming:, thanks for the fix :+1: